recipes/mpmath/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version="0.19" %}
+
+package:
+  name: mpmath
+  version: {{ version }}
+
+source:
+  fn: mpmath-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/m/mpmath/mpmath-{{ version }}.tar.gz
+  md5: af5cc956b2673b33a25c3e57299bae7b
+
+build:
+  number: 0
+  script: python -m pip install --no-deps .
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - mpmath
+    - mpmath.calculus
+    - mpmath.functions
+    - mpmath.libmp
+    - mpmath.matrices
+    - mpmath.tests
+
+about:
+  home: http://mpmath.org
+  license: BSD
```

Comment:
Should we do `license_family` here too? I don't care either way, but figured we might as well given the other one hase it.